app/views/staff/shared/activities/_activity.html.haml

```diff
@@ -424,7 +424,7 @@
       - if policy(activity_presenter).update? && step_is_complete_or_next?(activity: activity_presenter, step: :oda_eligibility)
         = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:oda_eligibility)}"), activity_step_path(activity_presenter, :oda_eligibility), t("summary.label.activity.oda_eligibility"))
 
-  - if @activity.is_project?
+  - if activity_presenter.is_project?
```

Comment:
I'd say not, because the presenter delegates to the object anyway. We shouldn't be using instance variables in partials anyway, because they should be able to stand on their own without relying on anything other than the locals that get set in the renderer 👍